packages/suite/src/views/firmware/FirmwareCustom.tsx

```diff
@@ -0,0 +1,155 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { ConfirmOnDevice } from '@trezor/components';
+import { useActions, useDevice, useFirmware } from '@suite-hooks';
+import { Translation, Modal } from '@suite-components';
+import { DeviceAcquire, DeviceUnknown, DeviceUnreadable } from '@suite-views';
+import * as routerActions from '@suite-actions/routerActions';
+import { TrezorDevice } from '@suite/types/suite';
+import { FirmwareInstallation, CloseButton } from '@firmware-components';
+import { ConnectDevicePromptManager, OnboardingStepBox } from '@suite/components/onboarding';
+import { useCachedDevice } from '@suite/hooks/firmware/useCachedDevice';
+import {
+    CheckSeedStep,
+    ReconnectDevicePrompt,
+    SelectCustomFirmware,
+} from '@suite/components/firmware';
```

Comment:
check `from '....'`

and import types using `import type ...`

Reply:
Fixed in [`5eec1822`](https://github.com/trezor/trezor-suite/commit/5eec1822f8e6504b0a2dc0ff955f37d141c0f921).